src/ceph_osd.cc

```diff
@@ -529,33 +529,30 @@ int main(int argc, const char **argv)
 				   client_byte_throttler.get(),
 				   nullptr);
   ms_public->set_policy(entity_name_t::TYPE_MON,
-                               Messenger::Policy::lossy_client(CEPH_FEATURE_UID |
-							       CEPH_FEATURE_PGID64 |
-							       CEPH_FEATURE_OSDENC));
+                        Messenger::Policy::lossy_client(osd_required));
   ms_public->set_policy(entity_name_t::TYPE_MGR,
-                               Messenger::Policy::lossy_client(CEPH_FEATURE_UID |
-							       CEPH_FEATURE_PGID64 |
-							       CEPH_FEATURE_OSDENC));
+                        Messenger::Policy::lossy_client(osd_required));
 
   //try to poison pill any OSD connections on the wrong address
   ms_public->set_policy(entity_name_t::TYPE_OSD,
 			Messenger::Policy::stateless_server(0));
 
   ms_cluster->set_default_policy(Messenger::Policy::stateless_server(0));
-  ms_cluster->set_policy(entity_name_t::TYPE_MON, Messenger::Policy::lossy_client(0));
+  ms_cluster->set_policy(entity_name_t::TYPE_MON,
+                         Messenger::Policy::lossy_client(0));
   ms_cluster->set_policy(entity_name_t::TYPE_OSD,
 			 Messenger::Policy::lossless_peer(osd_required));
   ms_cluster->set_policy(entity_name_t::TYPE_CLIENT,
 			 Messenger::Policy::stateless_server(0));
 
   ms_hb_front_client->set_policy(entity_name_t::TYPE_OSD,
-			  Messenger::Policy::lossy_client(0));
+                                 Messenger::Policy::lossy_client(0));
```

Comment:
i'd suggest avoid s/\tab/        / changes if possible.

Reply:
yeah, this is a pretty old file. And this kind of formatting nits will make it more difficult to backport:(

Reply:
OK, fixed it.